src/tnetdbg.cpp

```diff
@@ -121,3 +121,30 @@ std::string to_string(const std::wstring &input) {
 		ret += it;
 	return ret;
 }
+
+void print_simple_backtrace(size_t depth) {
+	using namespace backward;
+
+	StackTrace st; st.load_here(depth);
+
+	TraceResolver tr; tr.load_stacktrace(st);
+	for (size_t i = 0; i < st.size(); ++i) {
+		ResolvedTrace trace = tr.resolve(st[i]);
+		std::cout << "#" << i
```

Comment:
can't write to std::cout (stream could be in wide mode on windows and/or could needs that winapi way to use it)...